cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -943,6 +943,7 @@ def delete_image_on_dst(self):
         for image in images_to_delete:
             image_id = self.dst_cloud.get_image_id(image['name'])
             self.dst_cloud.glanceclient.images.delete(image_id)
+            self.dst_cloud.clean_tools.delete_image_by_id_from_db(image_id)
```

Comment:
@plomakin, please take a look at this line. Do we need it here? I think, we are deleting images on dst to check that new image will be created with new id.